src/pks/flow/constitutive_relations/wrm/wrm_van_genuchten.cc

```diff
@@ -205,5 +205,47 @@ void WRMVanGenuchten::InitializeFromPlist_() {
   }  
 };
 
+/* ******************************************************************
+* Suction formula: input is liquid saturation.
+****************************************************************** */
+double WRMVanGenuchten::suction_head(double s) {
+ 
+  double se = (s - sr_)/(1-sr_);
+  //if (function_ == FLOW_WRM_MUALEM) {
+  if (se > FLOW_WRM_TOLERANCE) {
+    return -(1./alpha_) * pow(pow(se, -1./m_) - 1, 1. - m_);
+  }else{
+    return -1e+10;
+  }
+
+ 
+}
+
+
+/* ******************************************************************
+ * D Relative permeability / D capillary pressure pc.
```

Comment:
fix the documentation.  Can you also add a comment on what "suction head" is?  I've never heard of it!